pkg/media/audiotrack.go

```diff
@@ -25,18 +25,11 @@ const (
 )
 
 type PCMLocalTrackParams struct {
-	WriteSilenceOnNoData bool
-	Encryptor            Encryptor
+	Encryptor Encryptor
 }
 
 type PCMLocalTrackOption func(*PCMLocalTrackParams)
 
-func WithWriteSilenceOnNoData(writeSilenceOnNoData bool) PCMLocalTrackOption {
-	return func(p *PCMLocalTrackParams) {
```

Comment:
Should we mark this deprecated instead of removing it? 

But, we're altering the behavior, so removing is probably better.

Reply:
Yeah, removing is better. There is only one user now. We can request them to remove that call. 

Will change README.